js/exporter/exceljs/export_pivot_grid.js

```diff
@@ -153,6 +157,10 @@ function _exportRow(rowIndex, cellCount, row, startColumnIndex, dataProvider, cu
     }
 }
 
+function _convertDateForExcelJS(date) {
+    return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours(), date.getMinutes(), date.getSeconds(), date.getMilliseconds()));
```

Comment:
https://github.com/DevExpress/DevExtreme/blob/20_2/js/exporter/exceljs/export_data_grid.js#L178 - the same function
'utils'?

Reply:
We have a separate task for allocation of general parts for DataGrid & PivotGrid export.